data/fm-info.json

```diff
@@ -1,5 +1,4 @@
-{
-    "scoringPeriodsFull":
+"scoringPeriodsFull":
```

Comment:
We're missing that opening bracket here for it to be valid. Do you mind updating?

Reply:
Fixed! 
(?)